File: solutions/p1.py
t = int(raw_input())

def isPrime(n):
    i = 2
    while i * i <= n:
        if n % i == 0:
            return False
        i += 1
    return True

for _ in xrange(0, t):
    n = int(raw_input()) + 1
    while not isPrime(n):
        n += 1
    print n

File: solutions/p1.java
import java.io.*;
import java.util.*;

public class p1 {
    public static boolean isPrime(int n) {
        for(int i = 2; i * i <= n; i++) {
            if(n % i == 0) return false;
        }
        return true;
    }
    
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        PrintWriter out = new PrintWriter(System.out);
        int t = in.nextInt();
        while(t-- > 0) {
            int n = in.nextInt();
            while(!isPrime(++n));
            out.println(n);
            out.flush();
        }
   }
}
